packages/nextjs/src/components/pages/oversikt-page/forms-list/panel/OversiktListPanel.tsx

```diff
@@ -15,7 +15,7 @@ import { ProductDetailsProps } from 'types/content-props/dynamic-page-props';
 import { ProductDetails } from 'components/_common/productDetails/ProductDetails';
 import { ProductDetailType } from 'types/content-props/product-details';
 import { IllustrationStatic } from 'components/_common/illustration/static/IllustrationStatic';
-import { LenkeStandalone } from 'components/_common/lenke/lenkeStandalone/LenkeStandalone';
+import { LenkeInline } from 'components/_common/lenke/lenkeInline/LenkeInline';
```

Comment:
Bytter ut lenkekomponent for å slippe unødvendig Aksel-styling som bare overstyres.